app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -93,17 +97,30 @@ int checkIfImageIsTooDark(BitmapRegionDecoder bitmapRegionDecoder) {
     }
 
     /**
-     * Pulls the pixels into an array and then runs through it while checking the brightness of each pixel.
```

Comment:
Let's keep the java doc for `checkIfImageIsDark` intact. It seems that it got deleted.

Reply:
Oh sorry, will do